docset/winserver2025-ps/smbshare/Reset-SmbServerConfiguration.md

```diff
@@ -143,6 +142,74 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuditClientCertificateAccess
```

Comment:
And here is where the Windows 11 behavior needs the version splitting - since this is new to Windows Server 2025 and Windows "vNext", these need some kind of note to distiguish "old windows 11" and new windows 11 (unless they rebrand it)

Reply:
@nedpyle, what behavior changes do we need to specify for this parameter? Does this parameter function differently in Win 11 22H2 to 23H2 (or above)?